app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CategoriesController < ApplicationController
+	def index
+		@categories_all = Category.all
+		# users = User.where(id: current_user.id)
+		# lessons = Lesson.where(user_id: current_user.id)
+		# num = lessons.pluck(:category_id)
+		@categories_done = current_user.categories
```

Comment:
@YuheiTsunoda Remove unnecessary code, and no need to create variable for `@categories_done`